components/dropdown/script.js

```diff
@@ -6,8 +6,10 @@
 
 import components from 'coherent-gameface-components';
 // eslint-disable-next-line no-unused-vars
-import ScrollableContainer from 'coherent-gameface-scrollable-container';
+import { ScrollableContainer } from 'coherent-gameface-scrollable-container';
 import template from './template.html';
+// eslint-disable-next-line no-unused-vars
+const scrollableContainer = ScrollableContainer;
```

Comment:
Why we need this variable here? :D it is not used anywhere

Reply:
Because I couldn't beat Webpack's optimization which prevents this from getting bundled - so I worked around it.